client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -951,6 +951,8 @@ private TxnSegments convert(CreateTxnResponse response) {
                                                       .setTxnId(ModelHelper.decode(txId));
             if (timestamp != null) {
                 txnRequest.setTimestamp(timestamp);
+            } else {
+                txnRequest.setTimestamp(Long.MIN_VALUE);
```

Comment:
How is this related to this PR?

Reply:
Commit from other PR got here as i accidentally built this branch on top of that one. 
Fixed it now.